### PR TITLE
toggle to show/hide small purchases

### LINKED ISSUE #1332: consider adding toggle to show/hide small purchases
The items tab has always been pretty crowded.

Consider adding a toggle that hides small items like TP scrolls and wards, and default it to on.

Example:
https://www.opendota.com/matches/3577473492/purchases

### PR DESCRIPTION
**Pull request by Argn0**
fixes #1332 

consumables in starting items are of interest I think

### CODE REVIEW

File: src/components/Match/matchColumns.jsx [hunk 1]
```diff
@@ -462,15 +462,17 @@ export const fantasyColumns = [
   },
 ].concat(fantasyComponents);
 
-export const purchaseTimesColumns = (match) => {
+export const purchaseTimesColumns = (match, showCommItems) => {
+  // IDs of common consumable items
+  const commCon = showCommItems ? [] : [46, 216, 44, 42, 43, 38, 257, 265, 188, 16];
```
Comment (howardchung) [inline src/components/Match/matchColumns.jsx]: I suggest using the names for greater readability

File: src/components/Match/matchColumns.jsx [hunk 2]
```diff
@@ -462,15 +462,17 @@ export const fantasyColumns = [
...
   const cols = [heroTdColumn];
   const bucket = 300;
   for (let i = 0; i < match.duration + bucket; i += bucket) {
     const curTime = i;
     cols.push({
       displayName: `${curTime / 60}'`,
       field: 'purchase_log',
-      displayFn: (row, column, field) =>
+      displayFn: (row, column, field) => [
```
Comment (howardchung) [inline src/components/Match/matchColumns.jsx]: what's the purpose of adding [ here?

File: src/components/Match/matchColumns.jsx (hunk 1 above)
Comment (Argn0) [reply src/components/Match/matchColumns.jsx]: shoul I put the array with item names outside the function?

Comment (howardchung) [reply src/components/Match/matchColumns.jsx]: sure you can put it in utility file if you think it might be reused

Comment (Argn0) [reply src/components/Match/matchColumns.jsx]: ok done

**Comment by howardchung** [comment]
check lint `npm run lint`

**Comment by Argn0** [comment]
ok I think I've fixed everything

File: src/utility/index.jsx [hunk 3]
```diff
@@ -592,3 +592,18 @@ export const wilsonScore = (up, down) => {
     1 + (z * z / n)
   );
 };
+
+export const commonConsumableItems = [
+  'Town Portal Scroll',
```
Comment (howardchung) [inline src/utility/index.jsx]: actually I mean the internal item name, like `tpscroll`, since those are unlikely to change.

Comment (Argn0) [reply src/utility/index.jsx]: items already had a qual property. I'm so blind :)

**Comment by howardchung** [comment]
Opinions on hiding/showing the starting items?  It might be confusing to the user that they don't count as consumables for display purposes

**Comment by Argn0** [comment]
Maybe we could rename time 0  to 'Starting Purchases'

**Comment by Argn0** [comment]
I remove consumables from starting items. I agree it was kinda confusing.

**Comment by Argn0** [comment]
I removed consumables from starting items, because I agree it was kinda confusing.

**Pull request merged**